tests/spectron/minimize-on-close.spectron.js

```diff
@@ -1,35 +1,31 @@
-const childProcess = require('child_process');
 const Application = require('./spectronSetup');
 const { isMac } = require('../../js/utils/misc');
 const constants = require('./spectronConstants');
+const robot = require('robotjs');
 
-let robot;
 let configPath;
 let app = new Application({});
 
 describe('Tests for Minimize on Close', () => {
 
     let originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 90000;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = Application.getTimeOut();
```

Comment:
what `Application.getTimeOut()` does exactly? and, where does it get the timeout information from?